gym/core.py

```diff
@@ -51,6 +51,7 @@ def __new__(cls, *args, **kwargs):
         env = super(Env, cls).__new__(cls)
         env._env_closer_id = env_closer.register(env)
         env._closed = False
+        env._spec = None
```

Comment:
I'm fine either way really, just think this might be a slight breaking change for people (I've run into code which sets .spec before, though it's usually a bug) without much benefit. I'd also have preferred that spec on wrapper is an attribute rather than property, but it's hard to do it due to when we actually set the spec.

So, if you'd like to make the change, just add something to the readme about this as a breaking change?